### PR TITLE
feat(how-to-propose-a-tutorial): upd to 2023 

### PR DESCRIPTION
**Pull request by tim08094495757**
## Types of changes

* **New feature**

## Description

update i18n, font-size, padding, structure
## Steps to Test This Pull Request

1. go to /speaking/tutorial
2. test languages (zh-hant, en-us)
3. test screen width (@min-width: 768px, 1024px, 1440px)

## Related Issue

#340

### CODE REVIEW

**Review by josix — CHANGES_REQUESTED** [review]
Thanks @tim08094495757 for the quick update. I would like to bring up a small issue as follows.
**issue:**  the length of the headers of fields seems to be too long, how about removing the "專業課程" from it. For example, these would be like "課程長度", "時間與場地", "售票政策", and "如何成為講師" ? WDYT?

File: pages/speaking/tutorial.vue [hunk 1]
```diff
@@ -117,5 +117,6 @@ export default {
     @media (min-width: 1024px) {
         @apply px-6;
     }
+    @apply xl:px-12;
```
Comment (tim08094495757) [inline pages/speaking/tutorial.vue:120]: I checked the design spec again, and found out that the padding in the larger screen is a little different from the page (/speaking/recording).

**Comment by tim08094495757** [comment]
> Thanks @tim08094495757 for the quick update. Everything looks great except a small issue as follows. **issue:** the length of the headers of fields seems to be too long, how about removing the "專業課程" from it. For example, these would be like "課程長度", "時間與場地", "售票政策", and "如何成為講師" ? WDYT?
> 
> English title could be updated as follows. "Duration", "Dates/Location", "Charging Policy", "How to become a instructor"

File: pages/speaking/tutorial.vue (hunk 1 above)
Comment (josix) [reply pages/speaking/tutorial.vue:120]: Nice, it looks great!

**Review by josix — APPROVED** [review]
LGTM! Thanks for the quick update.

**Pull request merged**